style.css

```diff
@@ -1,18 +1,752 @@
-/*@import "fonts.css";*/
-
-/*body {
-	font-family: "Montserrat-SemiBold";
-}*/
-
 .header, .start, .specifications, .salon, .description__title, .parameters__item, .safety__title, .stars__subtitle, .fivestars__textstars{
-	font-family: "Montserrat-SemiBold";
+font-family: "Montserrat-SemiBold";
 }
 .specifications__item, .description__link, .stars__item, .fivestars__text, .fivestars__link{
-	font-family: "Montserrat-Bold";
+font-family: "Montserrat-Bold";
 }
 .description__text, .safety__text{
-	font-family: "Montserrat-Medium";
+font-family: "Montserrat-Medium";
 }
 .parameters__small{
-	font-family: "Montserrat-Regular";
+font-family: "Montserrat-Regular";
+}
+
+
+body{
+max-width: 1440px;
+margin: 0 auto;
+display: flex;
+flex-direction: column;
+}
+
+.header{
+display: flex;
+max-width: 1440px;
+height: 51px;
+padding-top: 17px;
+justify-content: center;
+align-content: center;
+align-items: center;
+justify-content: space-between;
+}
+
+.menu{
+font-size: 1vw;
+display: flex;
+justify-content: center;
+align-content: center;
+}
+
+.menu__store{
+display: flex;
+justify-content: center;
+align-content: center;
+bottom: 20px;
+}
+
+.menu__link{
+min-width: 65px;
+display: flex;
+align-items: center;
+align-content: center;
+font-weight: 600;
+font-size: 15px;
+line-height: 18px;
+text-decoration: none;
+cursor: pointer;
+color: black;
+transition: color 0.3s ease-in;
+margin-right: 27px;
+}
+
+.menu__link_red{
+border-bottom: 2px solid red;
+display: flex;
+align-items: center;
+align-content: center;
+font-weight: 600;
+font-size: 15px;
+cursor: pointer;
+color: #E51937;
+margin-right: 63px;
+padding-bottom: 6px;
+padding-top: 6px;
+}
+
+.menu__link_roadster{
+margin-left: 20px;
+}
+
+.menu__link_energy{
+	margin-left: 15px;
+}
+
+.menu__link:hover{
+color: #E51937;
+}
+
+.header__burger{
+display: flex;
+align-items: center;
+justify-content: flex-start;
+bottom: 60px;
+width: 24px;
+height: 20px;
+cursor: pointer;
+margin-right: 3%;
+}
+
+.header__burger span{
+width: 24px;
+height: 4px;
+background-color: #E51937;
+}
+
+.header__burger span:before{
+position: absolute;
+content: " ";
+width: 24px;
+height: 4px;
+background-color: #E51937;
+transform: translateY(-9px);
+}
+
+.header__burger span:after{
+position: absolute;
+content: " ";
+width: 24px;
+height: 4px;
+background-color: #E51937;
+transform: translateY(9px);
+}
+
+
+section{
+display: flex;
+flex-direction: column;
+width: 100%;
+}
+
+.start{
+display: flex;
+align-items: center;
+}
+
+.start__title{
+display: flex;
+max-width: 370px;
+margin: 55px 0 0 0;
+font-style: normal;
+font-weight: 600;
+font-size: 90px;
+}
+
+.start__text{
+margin: 0px;
+font-style: normal;
+font-weight: 500;
+font-size: 15px;
+line-height: 18px;
+position: relative;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_rus3/pullRequest?prid=6047524)